lib/winapi/sysinfo.c

```diff
@@ -2,6 +2,28 @@
 #include <assert.h>
 #include <xboxkrnl/xboxkrnl.h>
 
+void GetSystemTime (LPSYSTEMTIME lpSystemTime)
+{
+    assert(lpSystemTime != NULL);
+
+    LARGE_INTEGER KTime;
+    TIME_FIELDS TimeFields;
```

Comment:
nit: Other code nearby appears to use `thatStyle` instead.

Reply:
Done